src/main/scala/no/ndla/crowdingateway/service/ConverterService.scala

```diff
@@ -10,17 +10,20 @@ package no.ndla.crowdingateway.service
 import no.ndla.crowdingateway.model.api.TranslationResponse
 import no.ndla.crowdingateway.model.domain.AddFileResponse
 
+import scala.util.{Success, Try}
+
 
 trait ConverterService {
   val converterService: ConverterService
 
   class ConverterService {
-    def asTranslationResponse(x: AddFileResponse): TranslationResponse = {
-      TranslationResponse(
+    def asTranslationResponse(x: AddFileResponse): Try[TranslationResponse] = {
```

Comment:
hmm, trenger denne returnere en `Try` hvis den alltid returnerer en `Success`? Hvis den brukes i en for-comprehension ett sted er det kanskje ryddigere å wrappe kallet rundt en `Success`?